Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[workspace]
+resolver = "2"
+members = [
+    "battery-service",
+    "cfu-service",
+    "embedded-service",
+    "espi-service",
+    "hid-service",
+    "power-button-service",
+    "power-policy-service",
+    "storage-bus",
+    "type-c-service",
+]
+exclude = ["examples/*"]
+
+[workspace.package]
+version = "0.1.0"
+edition = "2021"
+license = "MIT"
+repository = "https://github.com/OpenDevicePartnership/embedded-services"
+
+[workspace.dependencies]
+bitfield = "0.17.0"
+bitflags = "2.8.0"
+cfg-if = "1.0.0"
+chrono = { version = "0.4", default-features = false }
+cortex-m = "0.7.6"
+cortex-m-rt = "0.7.5"
+critical-section = "1.1"
+defmt = "0.3"
+document-features = "0.2.7"
+embassy-executor = { git = "https://github.com/embassy-rs/embassy" }
+embassy-futures = { git = "https://github.com/embassy-rs/embassy" }
+embassy-imxrt = { git = "https://github.com/OpenDevicePartnership/embassy-imxrt" }
+embassy-sync = { git = "https://github.com/embassy-rs/embassy" }
+embassy-time = { git = "https://github.com/embassy-rs/embassy" }
+embassy-time-driver = { git = "https://github.com/embassy-rs/embassy" }
+embedded-batteries-async = { git = "https://github.com/OpenDevicePartnership/embedded-batteries" }
+embedded-cfu-protocol = { git = "https://github.com/OpenDevicePartnership/embedded-cfu" }
+embedded-hal = "1.0"
+embedded-hal-async = "1.0"
+embedded-hal-nb = "1.0"
+embedded-io = "0.6.1"
+embedded-io-async = "0.6.1"
+embedded-services = { path = "./embedded-service" }
+embedded-storage = "0.3"
+embedded-storage-async = "0.4.1"
+embedded-usb-pd = { git = "https://github.com/OpenDevicePartnership/embedded-usb-pd", default-features = false }
+fixed = "1.23.1"
+heapless = "0.8.*"
+log = "0.4"
+postcard = "1.*"
+rand_core = "0.6.4"
+serde = { version = "1.0.*", default-features = false }
+tps6699x = { git = "https://github.com/asasine/tps6699x", branch = "remove-controller-trait" }
```

Comment:
One https://github.com/OpenDevicePartnership/tps6699x/pull/17 merges, this can be the default branch.